MadLibs

```diff
@@ -3,226 +3,85 @@ templ = random.randint(1, 3) #selecting 1 of 3 templates
 print(templ)
 print("Let's go! Enjoy playing with words!")
 if templ == 1:
-    Number = input("Please type a number")
```

Comment:
You could add a space before closing the quotation mark so that the users input wouldn't be attached to your question